README.md

```diff
@@ -31,6 +31,21 @@ steps:
     id: chipmunk_version
 ```
 
+Compare with the current github ref:
+
+```yaml
+steps:
+  - uses: oprypin/find-latest-tag@v1
+    with:
+      repository: docker/metadata-action
+      compares-to-latest: ${{ github.ref }}
+    id: compare
+  - run: |
+      echo "Current is newer: ${{ steps.compare.outputs.newer }}"
+      echo "Current is older: ${{ steps.compare.outputs.older }}"
+      echo "Current is equal: ${{ steps.compare.outputs.equal }}"
```

Comment:
How about an example that uses that output as a conditional (inside `if:`)? That seems more useful.